src/Module/Contact.php

```diff
@@ -664,30 +654,24 @@ public static function content($update = 0)
 			return $arr['output'];
 		}
 
-		$blocked = false;
-		$hidden = false;
-		$ignored = false;
-		$archived = false;
-		$all = false;
-
 		// @TODO: Replace with parameter from router
-		if (($a->argc == 2) && ($a->argv[1] === 'all')) {
```

Comment:
Just a question: So `contact/all` isn't a valid path anymore?

Reply:
No and I see where this is going, patch incoming.